package.json

```diff
@@ -43,6 +43,14 @@
     "typescript": "^3.8.2",
     "typings-tester": "^0.3.2"
   },
+  "peerDependencies": {
+    "immutable": "^4.0.0"
+  },
+  "peerDependenciesMeta": {
+    "immutable": {
+      "optional": true
+    }
+  },
```

Comment:
```suggestion
```
Let's just leave this out. As we are generally encouraging against using immutablejs, we should probably not mention it here.

Reply:
Yeah that makes sense. I definitely wasn't sure whether bringing in this dependency just for one type was a good idea or not. `unknown` is fine.